mscore/exportxml.cpp

```diff
@@ -3762,7 +3789,7 @@ static int findTrackForAnnotations(int track, Segment* seg)
       }
 
 //---------------------------------------------------------
-//  repeatAtMeasureStart -- write repeats at begin of measure
+//  repeatAtMeasureStart -ite repeats at begin of measure
```

Comment:
Got it. How do identify these before submitting a PR other than manually? My travis doesn't